src/chocolatey/infrastructure.app/commands/ChocolateyListCommand.cs

```diff
@@ -46,7 +46,7 @@ public virtual void configure_argument_parser(OptionSet optionSet, ChocolateyCon
                      "Source - Source location for install. Can use special 'webpi' or 'windowsfeatures' sources. Defaults to sources.",

```

Comment:
Maybe add something here to say when used with `--local-only` it is ignored?

Reply:
For what it's worth, as I said in the context above (perhaps it should have been in notes, instead), there seems to be no way to output that line without the modified line directly following it - e.g.
![image](https://user-images.githubusercontent.com/1975761/153592988-2bdec390-6025-4127-aa6e-4d527dc99444.png)

Do we need to say it twice in two lines if there's no way to display one without the other?